lib/ProductOpener/Import.pm

```diff
@@ -519,6 +519,14 @@ sub set_field_value (
 			$product_ref->{$field} = "";
 			delete $product_ref->{$field . "_tags"};
 		}
+
+		if (($field eq "allergens") or ($field eq "traces")) {
```

Comment:
You can use $options{replace_existing_values_when_importing_those_tags_fields} in this condition.